pkg/cephmgr/cephd/cephd.go

```diff
@@ -90,6 +90,24 @@ func GetCephdError(err int) error {
 	return cephdError(err)
 }
 
+// run a command in libcephd
+func RunCommand(command string, args []string) error {
+
+	// The command passes through args to the child process.  Look for the
+	// terminator arg, and pass through all args after that (without a terminator arg,
+	// FlagSet.Parse prints errors for args it doesn't recognize)
+	passthruIndex := 3
```

Comment:
or maybe we get rid of the pass through index and require passing the double dash as the indicator. I believe we're doing that most places already.